### PR TITLE
Update/add more UI updates/gh

### PR DESCRIPTION
**Pull request by Pseudoman21**
Preview:
image

### CODE REVIEW

**Comment by Pseudoman21** [comment]
Tig consolidate ko man po Kuya kay @crs26 si remaining balance, convenient daa po sa end niya para dae na po macommand para magcheck ki balance hehe kaya maski dae na daa po haliun hehe

**Comment by harveyjavier** [comment]
@Pseudoman21  pa check if applied na si ff. items:

- [x] Remove _Remaining faucet balance_ ([slack msg link](https://sparkpointio.slack.com/archives/C03548HLJ8K/p1681892849422669?thread_ts=1681891775.712079&cid=C03548HLJ8K))
- [x] Development environment error handling ([slack msg link](https://sparkpointio.slack.com/archives/C03548HLJ8K/p1681892912218899?thread_ts=1681891775.712079&cid=C03548HLJ8K))
- [x] Default color of active tab ([slack msg link](https://sparkpointio.slack.com/archives/C03548HLJ8K/p1681892992097329?thread_ts=1681891775.712079&cid=C03548HLJ8K))
- [x] Title: SparkLearn ICP Faucet, Favicon: SLETI Logo ([slack msg link](https://sparkpointio.slack.com/archives/C03548HLJ8K/p1681893125967029?thread_ts=1681891775.712079&cid=C03548HLJ8K))
- [x] _Canister ID_ only drop the _Wallet_ ([slack msg link](https://sparkpointio.slack.com/archives/C03548HLJ8K/p1681893385475529?thread_ts=1681891775.712079&cid=C03548HLJ8K))
- [x] Remove _(default)_ on ICP ([slack msg link](https://sparkpointio.slack.com/archives/C03548HLJ8K/p1681893473209359?thread_ts=1681891775.712079&cid=C03548HLJ8K))

**Comment by harveyjavier** [comment]
> Tig consolidate ko man po Kuya kay @crs26 si remaining balance, convenient daa po sa end niya para dae na po macommand para magcheck ki balance hehe kaya maski dae na daa po haliun hehe

🤔

**Comment by Pseudoman21** [comment]
> > Tig consolidate ko man po Kuya kay @crs26 si remaining balance, convenient daa po sa end niya para dae na po macommand para magcheck ki balance hehe kaya maski dae na daa po haliun hehe
> 
> 🤔

Pero ika po Kuya ano daw po mas maray.

**Comment by harveyjavier** [comment]
@sparklearnedtech/core much better ng di exposed si balance sa participants

**Comment by harveyjavier** [comment]
or pwede mag appear lang sa specific principal id. tingin nyo, @Pseudoman21 @crs26 ?

**Comment by Pseudoman21** [comment]
> or pwede mag appear lang sa specific principal id tingin nyo, @Pseudoman21 @crs26 ?

Maski remove nalang din daw po Kuya

**Comment by harveyjavier** [comment]
oks. pra less initial api call na din

**Comment by Pseudoman21** [comment]
> oks. pra less initial api call na din

Opo Kuya ✍️

File: src/component/Faucet/index.tsx [hunk 1]
```diff
@@ -18,7 +18,7 @@ export const Faucet: React.FC<FaucetProps> = ({inputRef, action, isLoading, succ
   return (
     <div className='row'>
         <div className='col-12 text-center'>
-            <div className='row d-flex justify-content-center'>
+            <div className='row d-flex justify-content-center mb-2'>
                 Remaining Balance: {balance}
```
Comment (harveyjavier) [inline src/component/Faucet/index.tsx]: Dae pa din na reremove

File: src/styles/globals.css [hunk 2]
```diff
@@ -27,5 +28,5 @@ a {
 
 .nav-tabs .nav-item.show .nav-link,
 .nav-tabs .nav-link.active {
-  background-color: #00ACED !important;
+  background-color: #95daf4 !important;
```
Comment (harveyjavier) [inline src/styles/globals.css:31]: @Pseudoman21 please see images. ang tiga mean kong default is yung wala ka ng gagalawin sa CSS. chineck mo ba reference? anyway inadd ko na lang dito image

image

image

**Review by harveyjavier — CHANGES_REQUESTED** [review]

File: src/styles/globals.css (hunk 2 above)
Comment (harveyjavier) [reply src/styles/globals.css:31]: > @Pseudoman21 please see images. ang tiga mean kong default is yung wala ka ng gagalawin sa CSS. chineck mo ba reference? anyway inadd ko na lang dito image
> 
> image
> 
> image

Comment (harveyjavier) [reply src/styles/globals.css:31]: image

Comment (Pseudoman21) [reply src/styles/globals.css:31]: Opo Kuya, removed na po ini didi po sa local ko dae ko pa lang po napush si changes . Si Chan po nag add ki background image diyan, bale tigpalitan ko nalang po kuta ki lighter color. Pero didi po sa isisync palang removed na po siya Kuya hehe

Comment (harveyjavier) [reply src/styles/globals.css:31]: @Pseudoman21 naga base kasi ako dito sa mga chineck mo. And nirun ko itong branch. So far itong mga _change request_ ko di naman applied yung mga checked sa checkbox

image

**Comment by Pseudoman21** [comment]
> @Pseudoman21 pa check if applied na si ff. items:
> 
> * [x]  Remove _Remaining faucet balance_ ([slack msg link](https://sparkpointio.slack.com/archives/C03548HLJ8K/p1681892849422669?thread_ts=1681891775.712079&cid=C03548HLJ8K))
> * [x]  Development environment error handling ([slack msg link](https://sparkpointio.slack.com/archives/C03548HLJ8K/p1681892912218899?thread_ts=1681891775.712079&cid=C03548HLJ8K))
> * [x]  Default color of active tab ([slack msg link](https://sparkpointio.slack.com/archives/C03548HLJ8K/p1681892992097329?thread_ts=1681891775.712079&cid=C03548HLJ8K))
> * [x]  Title: SparkLearn ICP Faucet, Favicon: SLETI Logo ([slack msg link](https://sparkpointio.slack.com/archives/C03548HLJ8K/p1681893125967029?thread_ts=1681891775.712079&cid=C03548HLJ8K))
> * [x]  _Canister ID_ only drop the _Wallet_ ([slack msg link](https://sparkpointio.slack.com/archives/C03548HLJ8K/p1681893385475529?thread_ts=1681891775.712079&cid=C03548HLJ8K))
> * [x]  Remove _(default)_ on ICP ([slack msg link](https://sparkpointio.slack.com/archives/C03548HLJ8K/p1681893473209359?thread_ts=1681891775.712079&cid=C03548HLJ8K))

Applied na po ito lahat Kuya sa checklist, kaso dae po masasimulate nguna sa local ta si claiming successful ta nakay Chan po naga run si dfx ta 😅

**Comment by harveyjavier** [comment]
no worries @Pseudoman21. di ko pa man tiga test si pag claim. UI pa lang. checking ako now

**Review by harveyjavier — CHANGES_REQUESTED** [review]
@Pseudoman21 ito din pala, drop the _Ledger_, _Account ID_ na lang. Thanks

image

**Comment by harveyjavier** [comment]
~@Pseudoman21 yung favicon di pa applied~ oks na pala ito, tig hard reload ko lang sa local end ko ✅ 
image

after hard reload:
image

**Review by harveyjavier — APPROVED** [review]

**Comment by Pseudoman21** [comment]
Thank you po Kuya 🚀

**Pull request merged**